app/views/play/CampaignView.coffee

```diff
@@ -1008,3 +1081,83 @@ module.exports = class CampaignView extends RootView
 
   onClickBrainPopReplayButton: ->
     api.users.resetProgress().then(=> document.location.reload())
+
+  getLevels: () ->
+    return @courseLevelsFake if @courseLevels?
```

Comment:
This helps us apply the existing nextLevel logic from the classroom on top of the world map.